### ISSUE TITLE
suggestion: add version info

### ISSUE THREAD

**Issue by kuchaguangjie** [application-research/estuary#123]
Add version number in code, or simply add tag as version in git commit, for easy version management.

**Comment by gmelodie** [comment]
Not sure we're on the same page, but I think you're talking about having releases with version numbers. As we don't have many milestones and stuff like that, I think it'd be hard to figure out when to release a new version. I could be wrong though, or we could be less strict on that. @whyrusleeping ideas here?

**Comment by jlogelin** [comment]
@corinne-antonia We are already deriving version numbers from the git tagging system. Currently, the build script uses the git tag to embed the [version in the binary](https://github.com/application-research/estuary/blob/master/Makefile#L51), so I think the semantic versions are aligned with the release versions.

tldr; we're all good :)

**Comment by corinne-antonia** [comment]
@jlogelin Can we close this then?

**Comment by jlogelin** [comment]
> @jlogelin Can we close this then?

yep
